modules/gdscript/tests/scripts/analyzer/warnings/unsafe_call_argument.gd

```diff
@@ -0,0 +1,42 @@
+func variant_func(x: Variant) -> void:
+	print(x)
+
+
+func int_func(x: int) -> void:
+	print(x)
+
+
+func float_func(x: float) -> void:
+	print(x)
+
+
+# We don't want to execute it because of errors, just analyze.
+func no_exec_test():
+	var untyped_int = 42
+	var untyped_string = "abc"
+	var variant_int: Variant = 42
+	var variant_string: Variant = "abc"
+	var typed_int: int = 42
+
+	variant_func(untyped_int) # No warning.
+	variant_func(untyped_string) # No warning.
+	variant_func(variant_int) # No warning.
+	variant_func(variant_string) # No warning.
+	variant_func(typed_int) # No warning.
+
+	int_func(untyped_int)
+	int_func(untyped_string)
+	int_func(variant_int)
+	int_func(variant_string)
+	int_func(typed_int) # No warning.
+
+	float_func(untyped_int)
+	float_func(untyped_string)
+	float_func(variant_int)
+	float_func(variant_string)
+	float_func(typed_int) # No warning.
```

Comment:
This works, and we would expect `int_func(typed_float)`, to generate an error from within the inner `if` that checks for "inverted" type compatibility, right? Not just a warning. Which is why that doesn't show up in this test suite?

Reply:
No, I just didn't add this test. As far as I remember, `int` (left) is compatible with `float` (right), but causes `NARROWING_CONVERSION` warning. Therefore, I think that there is no need for this test. The supertype and subtype are tested against `int` and `Variant`, I added `float_func` just to test for no false positive warning on compatible arguments.

Reply:
Just tested it and you are absolutely right - it does the narrowing conversion error so doesn't need to be tested here! :)